docs/mozilla-traffic-cop.rst

```diff
@@ -56,20 +61,44 @@ An implementation might look like::
         }
     });
 
-    cop.init();
+    eddie.init();
 
 In the above example, the string "exp_firefox_new_headline" will be used as the
 cookie name should a variation be chosen.
 
 The test will have 3 variations, each targeting 25% of users.
 
+There is also an optional configuration value to specify how long the cookie
+associated with a visitor for an individual experiment will last:
+``cookieExpires``. This value must be a ``Number`` and represents the number of
+hours the cookie will last. If omitted, the cookie will only last for the
+current browsing session.
```

Comment:
I ate some yogurt while looking out the window and thought about this. I think 24 hours is the right default. It's long enough that the user (hopefully) wouldn't be surprised (or even notice) if they got a different variation of a page, but not so long that they'll see the same variation for the life of the experiment.